src/main/java/bio/terra/common/Column.java

```diff
@@ -85,6 +85,39 @@ public Column type(TableDataType type) {
     return this;
   }
 
+  public boolean isTextType() {
+    switch (this.type) {
+      case TEXT:
+      case STRING:
+      case DIRREF:
+      case FILEREF:
+        return true;
+      default:
+        return false;
+    }
```

Comment:
You can use a switch expression here which makes the switch a bit more compact
```suggestion
    return switch (type) {
      case TEXT, STRING, DIRREF, FILEREF -> true;
      default -> false;
    };
```
Note that since this class has a no-arg constructor there's no way to know if `type` is has been initialized at this point, so this could result in an NPE.